nf_core/pipeline-template/lib/NfcoreTemplate.groovy

```diff
@@ -145,6 +145,61 @@ class NfcoreTemplate {
         output_tf.withWriter { w -> w << email_txt }
     }
 
+    //
+    // Construct and send adaptive card
+    // https://adaptivecards.io
+    //
+    public static void adaptivecard(workflow, params, summary_params, projectDir, log, multiqc_report=[]) {
```

Comment:
`log` and `multiqc_report` are not used, I think they can be removed, or even better add a log message if something fails.

Reply:
```suggestion
    public static void adaptivecard(workflow, params, summary_params, projectDir, log) {
```